portfolio/src/main/java/com/google/sps/servlets/NewTaskServlet.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for creating new tasks. */
+@WebServlet("/store-data")
+public class NewTaskServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String title = request.getParameter("text-input");
+    long timestamp = System.currentTimeMillis();
+    String color = request.getParameter("color");
+
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("title", title);
+    taskEntity.setProperty("timestamp", timestamp);
+    taskEntity.setProperty("description", color);
```

Comment:
I think that the temporary variables don't make the code more readable, so you could shorten it to:

```java
    taskEntity.setProperty("title", request.getParameter("text-input"));
    taskEntity.setProperty("timestamp", System.currentTimeMillis());
    taskEntity.setProperty("description", request.getParameter("color"));
```

On the other hand, if you adopt Maciej's suggestion and add some validation code, you will probably use the values more than once and then the temporary variables are useful.